paper.tex

```diff
@@ -231,9 +231,9 @@ \subsection{The subdivision density integral} \label{subsection:subdiv-density-i
 
 \subsection{Approximation of the subdivision density integral}
 
-The subdivision density integral (Section~\ref{subsection:subdiv-density-int}) is fairly straightforward to compute in the forward direction, but not invertible using a straightforward closed-form equation. Numerical techniques are possible, but require multiple iterations to achieve sufficient accuracy, so are slower. In this subsection, we present a straightforward and accurate approximation, constructed piecewise from easily invertible functions. If higher flattening quality is desired at the expense of slower computation, this approximation can be used to determine a good initial value for numeric techniques; two iterations of Newton solving are enough to refine this guess to within 32-bit floating point accuracy.
+The subdivision density integral (Section~\ref{subsection:subdiv-density-int}) is fairly straightforward to compute in the forward direction, but not invertible using a straightforward closed-form equation. Numerical techniques are possible, but require multiple iterations to achieve sufficient accuracy, so are slower. In this subsection, we present a straightforward and accurate approximation, constructed piecewise from easily invertible functions. This approximation was found empirically, by interactively tuning candiadate approximation functions in the Desmos graphing calculator. The exact integral and the approximation are shown in Figure~\ref{fig:espc}. Visually, it is clear that the agreement is close. We also built a test suite to exhaustively test the subdivision count using a property testing approach, finding that the worst case discrepancy between approximate and exact results is 6\%. If higher flattening quality is desired at the expense of slower computation, this approximation could be used to determine a good initial value for numeric techniques; two iterations of Newton solving are enough to refine this guess to within 32-bit floating point accuracy.
```

Comment:
This looks good. Maybe add a small parenthetical that says we are sharing the code for the test suite?:

"We also built a test suite _(included in our supplemental materials)_ to exhaustively..."